internal/management/repository/eventsourcing/handler/application.go

```diff
@@ -58,13 +58,7 @@ func (p *Application) Reduce(event *models.Event) (err error) {
 		}
 		return p.view.DeleteApplication(app.ID, event.Sequence)
 	case es_model.ProjectRemoved:
-		applications, err := p.view.ApplicationsByProjectID(event.AggregateID)
-		if err != nil {
-			logging.LogWithFields("HANDL-b2nET", "id", event.AggregateID).WithError(err).Warn("could not update existing projects")
-		}
-		for _, existing := range applications {
-			p.view.DeleteApplication(existing.ID, event.Sequence)
-		}
+		return p.view.DeleteApplicationsByPrjectID(event.AggregateID)
```

Comment:
Schriebfehler in Project
```suggestion
		return p.view.DeleteApplicationsByProjectID(event.AggregateID)
```